### PR TITLE
Add test for binary files on aarch64 platform

### LINKED ISSUE #335: Test for AArch64
There is no test for AArch64 now in dlk.

### PR DESCRIPTION
**Pull request by hadusam**
Close #335 
Add test for binary files on aarch64 platform (raspberry-pi).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by hadusam** [comment]
@iizukak @tsawada 
This PR is ready for review!
Please review it! Thanks 😄

File: tests/device_tests/test_device.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import unittest
+from nnlib import NNLib as NNLib
+
+
+class TestDevice(unittest.TestCase):
+    """Base class for Device Test."""
+
+
+    def get_param(self, test_case_path):
+        params = {
```
Comment (tsawada) [inline tests/device_tests/test_device.py]: return { ...}

File: tests/device_tests/test_device.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+# limitations under the License.
+# =============================================================================
+"""Test file for binary running on devices"""
+import numpy as np
+import os
+import unittest
+from nnlib import NNLib as NNLib
+
+
+class TestDevice(unittest.TestCase):
```
Comment (tsawada) [inline tests/device_tests/test_device.py]: I think `DeviceTest` is a better name

File: tests/device_tests/test_device.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    """Base class for Device Test."""
+
+
+    def get_param(self, test_case_path):
+        params = {
+            'library':os.path.join(test_case_path, "lib.so"),
```
Comment (tsawada) [inline tests/device_tests/test_device.py]: space after `:`

File: tests/device_tests/test_device.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""Test file for binary running on devices"""
+import numpy as np
+import os
```
Comment (tsawada) [inline tests/device_tests/test_device.py]: stdlib should come first.
Does this pass flake8?

File: tests/device_tests/test_device.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        params = {
+            'library':os.path.join(test_case_path, "lib.so"),
+            'input_npy': os.path.join(test_case_path, "input.npy"),
+            'expected_npy': os.path.join(test_case_path, "expected.npy"),
+        }
+
+        return params
+
+    def get_test_cases(self):
+        input_path = os.environ.get('DEVICE_TEST_INPUT_PATH')
```
Comment (tsawada) [inline tests/device_tests/test_device.py]: should fail gracefully when `DEVICE_TEST_INPUT_PATH` is not in env

File: tests/device_tests/test_device.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        test_case_paths = [d for d in os.listdir(input_path) if os.path.isdir(os.path.join(input_path,d))]
```
Comment (tsawada) [inline tests/device_tests/test_device.py]: i think it's better to use `pathlib` here (and maybe in the other part of this file)
```
import pathlib
p = pathlib.Path(input_path)
test_case_paths = [d for d in p.iterdir() if d.is_dir()]
```

File: tests/device_tests/test_device.py (hunk 1 above)
Comment (hadusam) [reply tests/device_tests/test_device.py]: Changed.

File: tests/device_tests/test_device.py (hunk 2 above)
Comment (hadusam) [reply tests/device_tests/test_device.py]: I think so too! Changed.

File: tests/device_tests/test_device.py (hunk 3 above)
Comment (hadusam) [reply tests/device_tests/test_device.py]: Fixed pep8 errors 🙇

File: tests/device_tests/test_device.py (hunk 4 above)
Comment (hadusam) [reply tests/device_tests/test_device.py]: Fixed pep8 errors including import-order 🙇

File: tests/device_tests/test_device.py (hunk 5 above)
Comment (hadusam) [reply tests/device_tests/test_device.py]: Changed it to raise KeyError when `DEVICE_TEST_INPUT_PATH` is not in env 👍

File: tests/device_tests/test_device.py (hunk 6 above)
Comment (hadusam) [reply tests/device_tests/test_device.py]: Changed to use `pathlib` 👍

**Comment by hadusam** [comment]
@tsawada 
Thanks for your review 😆  
I fixed them following your comments.
Please review it again! 👍

**Comment by iizukak** [comment]
Will be merged after test passed.

**Pull request merged**